networking/gcp/psa/examples/simple/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "region" {
+  description = "The GCP region to deploy the Artifact registry"
+  type        = string
+  default     = "europe-west9"
+}
+
+variable "project" {
+  description = "Project name"
+  type        = string
+  default     = "armonik-gcp-13469"
```

Comment:
This kind of information should not appear in repos

Reply:
[Same answer](https://github.com/aneoconsulting/ArmoniK.Infra/pull/75#discussion_r1303999794)